CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree iOS SDK Release Notes
 
+## unreleased
+* BraintreePayPalNativeCheckout (General Availability release)
```

Comment:
```suggestion
* BraintreePayPalNativeCheckout
```
We just added the GA note for the first GA release to make it easily searchable in the CHANGELOG